collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -2,10 +2,12 @@
 # Description: nvidia-smi netdata python.d module
 # Original Author: Steven Noonan (tycho)
 # Author: Ilya Mashchenko (ilyam8)
+# User Memory Stat Author: Guido Scatena (scatenag)
 
 import subprocess
 import threading
 import xml.etree.ElementTree as et
+import psutil
```

Comment:
there is no `psutil` in the py std lib

Reply:
Ok;
with https://github.com/netdata/netdata/pull/9372/commits/725bb737b4d4f0e54c17bf372191e701d8d4ab97 I removed dependency from psutil for os and pwd